openeo/rest/job.py

```diff
@@ -230,9 +262,13 @@ def soft_error(message: str):
             """.format(i=self.job_id)))
             # TODO: make it possible to disable printing logs automatically?
             # TODO: render logs jupyter-aware in a notebook context?
-            # TODO: only print the error level logs? Or the tail of the logs?
             print("Printing logs:")
-            print(self.logs())
+            print("Only printing messages of log level ERROR.")
+            print(
+                "To get logs for all log levels, use the method `BatchJob.logs` with the parameter `log_level` ."
+            )
+            print('For example: my_batch_job.logs(log_level="info")')
+            print(self.logs(log_level=logging.ERROR))
```

Comment:
5 `print` statements looks a bit much I think. The first two can be merged for example. 
Third and fourth can be combined more compactly too

Reply:
I tried to come up with a more concise message that is still as informative as possible, and bundled it in one cleaner print statement.
Let me know what you think.

Reply:
I merged the PR already  but then  noticed there is even more duplication with a couple of lines above.
did a follow up in 0ceced3ba685c0d789702b3c92e9de94e2692e9d